src/02-common-principles.md

```diff
@@ -540,16 +540,28 @@ For additional rules, see below:
 Describing dates and timestamps:
 
 -   Date time information MUST be expressed in the following format
-    `YYYY-MM-DDThh:mm:ss[.000000]` (year, month, day, hour (24h), minute,
-    second, and optionally fractional second).
-    This is equivalent to the RFC3339 "date-time" format, time zone is always
-    assumed as local time).
+    `YYYY-MM-DDThh:mm:ss[.000000][Z]` (year, month, day, hour (24h), minute,
+    second, optional fractional seconds, and optional UTC time indicator).
+    This is almost equivalent to the [RFC3339](https://tools.ietf.org/html/rfc3339)
+    "date-time" format, with the exception that UTC indicator `Z` is optional and
+    non-zero UTC offsets are not indicated.
+    If `Z` is not indicated, time zone is always assumed to be the local time of the
+    dataset viewer.
```

Comment:
Should this be changed to "dataset generator" or is assuming viewer fine?

Reply:
I'd say dataset generator, but I don't feel strongly about it (which is why I simply accepted @effigies suggestion to make it "viewer")

Reply:
Fair enough. I think it's fine either way since most people would be most concerned with circadian rhythms or time between scans, and that information is conveyed no matter what.

Reply:
I don't feel strongly about what SHOULD be done, but just to explain my reasoning, when a compliant parser interprets the data, it will depend on its current locale, and is thus the viewer's local time. Using offsets to indicate the generator's local time is the ISO-8601 mechanism for that, and we have foreclosed that possibility. So you get local time of reader or UTC, depending on what the dataset curator thinks is more important.